apps/minifront/src/components/v2/get-v2-link.ts

```diff
@@ -0,0 +1,4 @@
+import type { PagePath } from '../metadata/paths.ts';
+
+/** @todo: Remove this function and its uses after we switch to v2 layout */
+export const getV2Link = (path: PagePath) => `/v2${path}`;
```

Comment:
thought: maybe instead of manipulating strings to create paths, we should rely on utilities provided by the router https://reactrouter.com/en/main/hooks/use-resolved-path

not a task for this PR